cmd/snap/cmd_snap_op.go

```diff
@@ -211,6 +212,28 @@ func (x *channelMixin) setChannelFromCommandline() error {
 	return nil
 }
 
+// show what has been done
+func showDone(names []string, op string) error {
+	cli := Client()
+	snaps, err := cli.List(names)
+	if err != nil {
+		return err
+	}
+
+	for _, snap := range snaps {
+		channelStr := ""
+		if snap.Channel != "" {
+			channelStr = fmt.Sprintf(" (%s)", snap.Channel)
+		}
+		developerStr := ""
+		if snap.Developer != "" {
+			developerStr = fmt.Sprintf(" from '%s'", snap.Developer)
+		}
+		fmt.Fprintf(Stdout, "%s%s %s%s %s\n", snap.Name, channelStr, snap.Version, developerStr, op)
```

Comment:
Thanks! I should have comments this in the code. I was/am aware of this but ignored it for now, this comment is a good reason to fix it though :)